src/components/CommentForm.js

```diff
@@ -11,6 +11,14 @@ import TagDescriptionEditor from './TagDescriptionEditor'
 import cx from 'classnames'
 var { array, bool, func, object, string } = React.PropTypes
 
+// The interval between repeated typing notifications to the web socket. We send
+// repeated notifications to make sure that a user gets notified even if they
+// load a comment thread after someone else has already started typing.
+const STARTED_TYPING_INTERVAL = 5000
```

Comment:
i increased these time limits to avoid the possibility of "flicker" if a person is taking their time typing a comment